src/frequenz/client/reporting/_client.py

```diff
@@ -252,3 +253,33 @@ def dt2ts(dt: datetime) -> PBTimestamp:
         except grpcaio.AioRpcError as e:
             print(f"RPC failed: {e}")
             return
+
+    @staticmethod
+    def calculate_energy_metric(
```

Comment:
Needs a clearer name, maybe `cumulative_energy`, `total_energy, `integrated_power` or so, depending on which approach we go for.